### ISSUE TITLE
slow test speed

### ISSUE THREAD

**Issue by shenshenzhanzhan** [yingcong/Facelet_Bank#4]
Is it normal to test slowly? I have used GPUs，A picture feels like it takes five minutes。

**Comment by yingcong** [comment]
No, it is not normal. Usually with a GPU it should take less than 1 second. Is the tested photo very large?

**Comment by shenshenzhanzhan** [comment]
I just use your test photo, it needs serveral minutes to load encoder and decoder weights . The tested photo also needs serveral minutes.

**Comment by shenshenzhanzhan** [comment]
I just use your test photo, it needs serveral minutes to load encoder and decoder weights . The tested photo also needs serveral minutes.

**Comment by yingcong** [comment]
Seems strange... I wonder if the file I/O is slow? Loading weights is also usually fast except for the first run.

**Comment by shenshenzhanzhan** [comment]
Maybe, but when i uses the cpu(about 10s), it seems more quickly than gpu version.
The GPU vesion spends a lot time after printing "loading pretrained weights of VGG decoder", so why?
